public/style.css

```diff
@@ -1,3 +1,14 @@
+@import url('https://fonts.googleapis.com/css2?family=Asap:wght@400;700&family=Nunito:wght@300;400;700&family=Poppins:wght@400;700&display=swap');
```

Comment:
I see that you included three font families:
- family=Asap:wght@400;700
- family=Nunito:wght@300;400;700
- family=Poppins:wght@400;

But you only used Poppins. 
Loading fonts is 'heavy' on the browser and takes a rather great part of the loading time.
Removing unused fonts is an easy fix 😄